js/privatebin.js

```diff
@@ -4914,6 +5158,13 @@ jQuery.PrivateBin = (function($, RawDeflate) {
 
             DOMPurify.setConfig({SAFE_FOR_JQUERY: true});
 
+            // center all modals
+            $('.modal').on('show.bs.modal', function(e) {
```

Comment:
Uhm, why can't you use CSS to apply this effect? Would be the better code style likely, instead of doing stylistic things via JS this way…

Reply:
Unfortunately this is not possible as by design bootstrap.js call the jQuery `$element.show()`
Dynamically:
https://github.com/twbs/bootstrap/blob/v3.3.7/js/modal.js#L84

In jQuery:
https://github.com/jquery/jquery/blob/3.3.1/src/css/showHide.js#L27

The getDefaultDisplay will return `block` instead of `flex`.

This cause an undesired out come as following:
If css is applied when the modal is open only given the following selector:
```
.modal.in {
  display: flex;
}
```
The transition of the element will be broken when the element disappear, as bootstrap.js remove the `in` class first before the transition start, causing the element to jump to top.

It is very hard to work with legacy technology while trying to tackle modern problems...